README.md

```diff
@@ -237,14 +237,25 @@ To receive push notifications, follow the steps above and run the app on a physi
 
 ## Building with Firebase on Apple platforms
 
-Firebase 8.9.0 introduced official beta support for macOS, Catalyst, and tvOS. watchOS continues
-to be community supported. Thanks to community contributions for many of the multi-platform PRs.
+Firebase 8.9.0 introduced official beta support for macOS, Catalyst, and tvOS. visionOS and watchOS
```

Comment:
This is a bit confusing.

The first part of the sentence mentions Firebase 8.9.0, which adds macOS, Catalyst, and tvOS. It goes on to talk about visionOS.

I think we should write "In Firebase 10.xx.x, we added support for visionOS. visionOS and watchOS are community-supported...